### DISCUSSION TITLE
osu!catch's total hit just count Fruits but not Fruit trails

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#29328] [Q&A] [answered]
in stable, Fruit trails count as 50 and 100, and will count as a hit to total hit
in lazer, Fruit trails and Fruit are recorded separately and will not count a hit

This can be proven by comparing user data

see https://osu.ppy.sh/wiki/en/Game_mode/osu!catch#gameplay-basics

**Comment by bdach** [comment]
Which "hit total" are you referring to here? The one on user profile? Something else entirely?

**Reply by cdwcgt** [reply]
![image](https://github.com/user-attachments/assets/63253658-4d29-4331-8fe2-2648c1caa55e)

this is the statistical information

Sorry the description is not clear, I think the total hit is only the one with the profile

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-queue-score-statistics/pull/278
